spec/requests/quotes_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
```

Comment:
I don't think that for this challenge you need this file. We are really just writing a feature test. We are calling an api but we will typically write requests when we are exposing an endpoint.